packages/css/src/components/tabs/tabs.scss

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../common/breakpoint";
+
+.amsterdam-tabs-underline {
+  bottom: 0;
+  content: "";
+  display: block;
+  height: var(--amsterdam-tabs-border-width);
+  left: 0;
+  position: absolute;
+  right: 0;
+  width: auto;
+}
+
+.amsterdam-tabs__list {
+  border-bottom: var(--amsterdam-tabs-border-width) solid var(--amsterdam-tabs-border-color);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  overflow-x: auto;
+  position: relative;
+
+  @media screen and (min-width: $amsterdam-breakpoint-wide) {
+    overflow: visible;
+  }
```

Comment:
This is to allow the focus ring to be visible on desktop. On mobile the overflow is set to auto (to allow hozirzontal scroll) and since focus is not really a thing on touchscreens I'd allow it.

Reply:
Hmm, tricky. Focus _is_ a thing on touch screens, e.g. for people using a keyboard to navigate on their phone.

However, we won’t get the combination of scrolling and a visible focus outline through `overflow: scroll visible` because “Setting one axis to visible while setting the other to a different value results in visible behaving as auto.” which removes the scrolling feature.

Reply:
I think a common fix for this is to place the focus for tab buttons **inside** the button, [like they did here](https://react-spectrum.adobe.com/react-aria/Tabs.html). It's a bit of a departure from our current focus pattern, but it's a robust fix I think

Reply:
Good one, please check the solution in the browser. We might need to implement the focus ring as designed in Figma now.